Lib/test/test_typing.py

```diff
@@ -3738,6 +3738,18 @@ def test_repr(self):
 
         self.assertEqual(repr(UserId), f'{__name__}.UserId')
 
+    def test_pickle(self):
+        UserId = NewType('UserId', int)
+
+        s = pickle.dumps(UserId)
+        loaded = pickle.loads(s)
+
+        self.assertEqual(loaded.__supertype__, int)
+        self.assertEqual(loaded.__name__, 'UserId')
+        self.assertEqual(loaded.__qualname__, 'UserId')
```

Comment:
Could you please use test case with `__qualname__` != `__name__`?

Reply:
Fixed, thanks